config.example.py

```diff
@@ -6,5 +8,9 @@
         "port": "5432",
         "database": "my-database",
```

Comment:
database: 'my-database' - is it my changes?

Reply:
That's just the placeholder for the database schema name (as in https://github.com/elsehow/taaraxtak/blob/main/config.example.py#L7). You should replace `my-database` with whatever your database schema is called where you installed it.